src/test/java/de/tum/in/www1/artemis/exam/ExamUserIntegrationTest.java

```diff
@@ -201,17 +195,24 @@ void testVerifyExamUserAttendance() throws Exception {
         headers.set("X-Artemis-Client-Fingerprint", "bar");
         headers.set("X-Forwarded-For", "10.0.28.1");
 
-        for (var studentExam : studentExams) {
+        studentExams.parallelStream().forEach(studentExam -> {
             var user = studentExam.getUser();
             database.changeUser(user.getLogin());
-            var response = request.get("/api/courses/" + course2.getId() + "/exams/" + exam2.getId() + "/student-exams/" + studentExam.getId() + "/conduction", HttpStatus.OK,
-                    StudentExam.class, headers);
+            StudentExam response;
+            try {
+                response = request.get("/api/courses/" + course2.getId() + "/exams/" + exam2.getId() + "/student-exams/" + studentExam.getId() + "/conduction", HttpStatus.OK,
```

Comment:
Sinne you're already refactoring and we have a style issue, could you either remove the import or change all occurrences to only `OK`

Reply:
Noted!